tests/test_journal.py

```diff
@@ -345,12 +348,24 @@ def test_submission(self, journal, openreview_client, test_client, helpers):
 note={Under review}
 }'''
 
+        ## try to make an assignment before the scores were computed
+        with pytest.raises(openreview.OpenReviewException, match=r'Can not add assignment, invitation is not active yet.'):
```

Comment:
I notice that this line fails randomly that the post of edge is successful
I tried to assert False before this line to manually post edge it's also successful